ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -242,6 +244,11 @@ descriptor:
   yaxis: "Horizontal axis of a three-dimensional Cartesian coordinate system (ordinate)."
   zaxis: "Vertical axis of a three-dimensional Cartesian coordinate system (applicate)."
   zone: "Region of building which is conditioned."
+  #New subfields for DE-FRA-NEM32
+  color: "Color of light or of a physical medium"
+  cie: "cie light color schema"
```

Comment:
CIE must be uppercase in the description string

Reply:
...done